decidim-assemblies/lib/decidim/api/assembly_type.rb

```diff
@@ -13,49 +13,49 @@ class AssemblyType < Decidim::Api::Types::BaseObject
 
       description "An assembly"
 
-      field :id, ID, "The internal ID for this assembly", null: false
-      field :subtitle, Decidim::Core::TranslatedFieldType, "The subtitle of this assembly", null: true
-      field :short_description, Decidim::Core::TranslatedFieldType, "The sort description of this assembly", null: true
       field :description, Decidim::Core::TranslatedFieldType, "The description of this assembly", null: true
-      field :slug, String, "The slug of this assembly", null: false
       field :hashtag, String, "The hashtag for this assembly", null: true
+      field :id, ID, "The internal ID for this assembly", null: false
       field :published_at, Decidim::Core::DateTimeType, "The time this assembly was published", null: false
       field :reference, String, "Reference for this assembly", null: false
+      field :short_description, Decidim::Core::TranslatedFieldType, "The sort description of this assembly", null: true
+      field :slug, String, "The slug of this assembly", null: false
+      field :subtitle, Decidim::Core::TranslatedFieldType, "The subtitle of this assembly", null: true
 
```

Comment:
Is there any reason why we have two blocks of text of fields on this case? Shouldn't they all be the same block ordered alphabetically?